nautobot/dcim/tests/test_views.py

```diff
@@ -312,10 +312,10 @@ def setUpTestData(cls):
         rackroles = Role.objects.get_for_model(Rack)[:2]
 
         statuses = Status.objects.get_for_model(Rack)
-        cls.status_active = statuses.get(slug="active")
+        cls.status = statuses[0]
 
         cable_statuses = Status.objects.get_for_model(Cable)
-        cls.cable_connected = cable_statuses.get(slug="connected")
+        cls.cable_connected = cable_statuses[1]
```

Comment:
Might need to keep this as-is until `Connected` stops being a magic status for Cables.